src/backend/distributed/metadata/node_metadata.c

```diff
@@ -507,7 +507,13 @@ citus_disable_node(PG_FUNCTION_ARGS)
 {
 	text *nodeNameText = PG_GETARG_TEXT_P(0);
 	int32 nodePort = PG_GETARG_INT32(1);
-	bool synchronousDisableNode = PG_GETARG_BOOL(2);
+
+	bool synchronousDisableNode = 1;
```

Comment:
I don't understand how master_disable_node() should work (see also #2063 on this topic), but if synchronousDisableNode = 0 by default, some tests are failing. For example, isolation_add_remove_node with
```
 step s1-disable-node-1: 
  SELECT 1 FROM master_disable_node('localhost', 57637);
  SELECT public.wait_until_metadata_sync();
 
-?column?
----------------------------------------------------------------------
-       1
-(1 row)
-
-wait_until_metadata_sync
----------------------------------------------------------------------
-
-(1 row)
-
+ERROR:  disabling the first worker node in the metadata is not allowed
 step s2-disable-node-1: 
  SELECT 1 FROM master_disable_node('localhost', 57637);
  SELECT public.wait_until_metadata_sync();
- <waiting ...>
+
+ERROR:  disabling the first worker node in the metadata is not allowed
```
in diff